eng/config/PublishData.json

```diff
@@ -54,12 +54,21 @@
       "vsBranch": "master",
       "vsMajorVersion": 16
     },
+    "release/dev16.4-preview2-vs-deps": {
+      "nugetKind": ["Shipping", "NonShipping"],
+      "version": "3.4.*",
+      "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+      "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+      "channels": [ "dev16.4p2" ],
+      "vsBranch": "master",
```

Comment:
@RikkiGibson This doesn't look right. Should be rel/d16.4 or something like that?

Reply:
Good catch, will submit a followup PR to fix.